test/e2e/webdriver/chrome.js

```diff
@@ -110,6 +110,7 @@ class ChromeDriver {
     return {
       driver,
       extensionUrl: `chrome-extension://${extensionId}`,
+      extensionId,
```

Comment:
the code already assumed this existed in `buildWebDriver`

<img width="714" alt="image" src="https://github.com/user-attachments/assets/e39b6d79-0c30-45e1-9946-3a2963b69560">